Formula/dagger.rb

```diff
@@ -0,0 +1,40 @@
+class Dagger < Formula
+  desc "Portable devkit for CI/CD pipelines"
+  homepage "https://dagger.io"
+  url "https://github.com/dagger/dagger.git",
+      tag:      "v0.2.4",
+      revision: "b32c8732bc7bd932dbdb5dc42fe2434c53cfeb38"
+  license "Apache-2.0"
+
```

Comment:
I think we can also include head build as well.

Reply:
done!